python/ray/_raylet.pyx

```diff
@@ -593,6 +593,82 @@ cdef store_task_errors(
         raise RayActorError.from_task_error(failure_object)
     return num_errors_stored
 
+cdef execute_dynamic_generator_task(
```

Comment:
Hmm it is actually the "execution" of the task though. I actually changed the name to execute because I thought it would be less confusing :D

Maybe "execute_dynamic_generator_task_and_store_outputs"?

Reply:
Sure. Or maybe `execute_dynamic_generator_and_store_outputs` to avoid the confusion that the task is fully executed inside this function.